test/access-token.test.js

```diff
@@ -249,6 +249,39 @@ describe('loopback.token(options)', function() {
         });
     });
 
+    it('should overwrite invalid existing token (is !== undefined and has no "id" property) ' +
+      ' when enableDoubkecheck is true',
+    function(done) {
+      var token = this.token;
+
+      app.use(function(req, res, next) {
+        req.accessToken = null;
+        next();
+      });
+
+      app.use(loopback.token({
+        model: Token,
+        enableDoublecheck: true,
+      }));
+
+      app.get('/', function(req, res, next) {
+        res.send(req.accessToken);
+      });
+
+      request(app).get('/')
+        .set('Authorization', token.id)
+        .expect(200)
+        .end(function(err, res) {
+          if (err) return done(err);
+          expect(res.body).to.eql({
```

Comment:
Is it necessary to check all token properties here? I am worried that if we add a new Token property in the future, we will have to fix way too many tests that don't really depend on specific Token properties.

It think it is enough to check that `res.body.id` is the same as `token.id`, isn't it?